internal/utils/kruize/kruize_api.go

```diff
@@ -196,3 +196,32 @@ func Is_valid_recommendation(recommendation kruizePayload.Recommendation, experi
 		return false
 	}
 }
+
+func Delete_experiment_from_kruize(experiment_name string) {
+	url := cfg.KruizeUrl + "/createExperiment"
+	data := []map[string]string{
+		{"experiment_name": experiment_name},
+	}
+	payload, err := json.Marshal(data)
+	if err != nil {
+		log.Error("unable to marshal payload to json: ", err)
+		return
+	}
+
+	client := &http.Client{}
+	req, err := http.NewRequest("DELETE", url, bytes.NewBuffer(payload))
+	if err != nil {
+		log.Error("unable to create request object ", err)
+		return
+	}
+	res, err := client.Do(req)
+	if err != nil {
+		kruizeAPIException.WithLabelValues("/deleteExperiment").Inc()
+		log.Error("error Occured while deleting experiment: ", err)
+		return
+	}
+	defer res.Body.Close()
+	if res.StatusCode == 201 {
```

Comment:
The generic search says,
~~~
If the item is was successfully deleted, return 200 (the resource exists as of the time of the request, and the action was executed). If it's being queued for execution, return 202. If the resource has already been deleted, return 410. If it otherwise doesn't exist, return 404.
~~~

If I see the kruize docs then they are returning 201 however are there chances of 200/202/410/404?

Reply:
Apart from `201` kruize does send `400` if it fails to delete the experiment. I will add the else block to increase prometheus counter and to log error.